src/pages/Agenda/Agenda.scss

```diff
@@ -60,16 +60,42 @@
 }
 
 .agenda-page__content__banner__text {
-  font-family: "Inter";
-  font-weight: 700;
-  font-size: 48px;
   position: absolute;
-  color: white;
   width: 100%;
   height: 100%;
   display: flex;
+  flex-direction: column;
   justify-content: center;
   align-items: center;
+  border-radius: 16px;
+  background-color: #00000030;
+}
```

Comment:
That background-color does not look right (I mean the code)
Why is it 8 char long?